### ISSUE TITLE
Prototype equip doesn't work for ring

### ISSUE THREAD

**Issue by ChefKeeper** [blizzhackers/kolbot#77]
Trying out the new prototype to equip item from [this PR](https://github.com/blizzhackers/kolbot/pull/14)

added these lines in Town.js doChores function : 
```
this.move('stash') && this.openStash();
delay(100);
let naturePeace = me.getItems(-1).filter(item => item.getStat(sdk.stats.Restinpeace)).first();
print("found: [" + naturePeace.fname + "]");
let oldItems = naturePeace.equip();//sdk.body.RingLeft);
delay(500);
me.cancel(); // stash might still be open
print("close dat stash");
delay(500);
me.cancel(); // stash might still be open
print("close dat stash");
delay(500);
Skill.cast(58); // cast energy shield
delay(500);
me.overhead("Wearing nature's peace");
delay(500);
Town.openStash();
delay(500);
//oldItems.rollback(); // put old items back on
delay(500);
print("END Wearing other ring again");
me.cancel(); // stash might still be open
delay(500);
```

It does succesfully find my nature's peace from stash and click on it, but after doesn't equip it.
Also tryed with a memory staff in stash, it does take of my shield take the staff but doesn't equip.
Same if nothing equiped at the moment, generally speaking looks like **bot isn't able to equip anything** 😢

**Comment by ChefKeeper** [comment]
Did some digging, might come up with a fix.

However there's a bigger issue under the hood where `me.getItems(-1)` would get the wrong `bodylocation` for 1 item in stash saying character was wearing it :
I had memory and Nature's peace in stash, testing to swap any weared ring with nature's peace, it would flag as the memory being weared (as a ring ?)
When I droped memory on ground all is working.

Bottom line : `bodylocation` **might bug**, we should keep that in mind when scripting.
